TestApp/update-npm-packages.sh

```diff
@@ -24,5 +24,5 @@ npm install
 echo "Build shared framework..."
 (cd ../AppCenterReactNativeShared/ios && SRCROOT=`pwd` ./build-fat-framework.sh)
 
-echo "Running jetify for react-native 0.60.0-rc3..."
+echo "Running jetify for react-native 0.60.0..."
```

Comment:
We will update RN and forget to update this echo line. Can we remove version or why we do it?
```suggestion
echo "Running jetify"
```